layouts/partials/head.html

```diff
@@ -48,6 +48,11 @@
 <script src="{{ "js/bootstrap-4.3.1.min.js" | relURL }}"></script>
 <script src="{{ "js/sweetalert-2.1.2.min.js" | relURL }}"></script>
 
+{{ if eq .Params.mermaid true }}
+<!-- Copied from https://unpkg.com/mermaid@8.5.0/dist/mermaid.min.js -->
+<script async src="{{ "js/mermaid.min.js" | relURL }}"></script>
+{{ end }}
```

Comment:
@jimangel , Question:
Do you know how the field `.Params.js` is used?

```
# config.toml
js = [
  "custom-jekyll/tags",
  "script"
]
```

Reply:
I have no clue. It looks like it does the same thing twice (explicitly first then in shortcode after:
```
<script src="{{ "js/script.js" | relURL }}"></script>
<script src="{{ "js/custom-jekyll/tags.js" | relURL }}"></script>
{{ with .Params.js }}{{ range (split . ",") }}<script src="{{ (trim . " ") | relURL }}"></script><!-- custom js added -->
```